File: GitHubPop/app/src/main/java/com/igormeira/githubpop/model/Repositories.java
package com.igormeira.githubpop.model;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class Repositories {

    @SerializedName("items")
    private List<Repository> repositories;

    public List<Repository> getRepositories() {
        return repositories;
    }

    public void setRepositories(List<Repository> repositories) {
        this.repositories = repositories;
    }
}

File: app/src/main/java/br/edu/ifsp/scl/desafio_android/model/Repositories.kt
package br.edu.ifsp.scl.desafio_android.model

import com.google.gson.annotations.SerializedName

data class Repositories (
    @SerializedName("total_count") var total_count: Long,
    @SerializedName("incomplete_results") var incomplete_results: Boolean,
    @SerializedName("items") var items: LinkedHashSet<Repository?>? = linkedSetOf()
)